main.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+Dir["#{File.dirname(__FILE__)}/lib/**/*.rb"].sort.each { |file| require file }
```

Comment:
Можно поменять на:
```suggestion
Dir["#{File.dirname(__FILE__)}/lib/**/*.rb"].sort.each(&method(:require)
```
не уверен как лучше :slightly_smiling_face: